db/migrate/20230103152832_reassign_collecting_signature_petitions_to_under_consideration.rb

```diff
@@ -0,0 +1,23 @@
+class ReassignCollectingSignaturePetitionsToUnderConsideration < ActiveRecord::Migration[6.1]
+  def up
+    execute(
+      "UPDATE petitions
+      SET referred_at = open_at, referral_threshold_reached_at = open_at
+      WHERE referred_at IS NULL
+      AND open_at IS NOT NULL;"
+    )
+
+    execute(
+      "UPDATE petitions
+      SET collect_signatures = true
+      WHERE (state = 'pending'
+        OR state = 'validated'
+        OR state = 'sponsored'
+        OR state = 'flagged'
+        OR state = 'opened')
+      AND collect_signatures = false"
+    )
+  end
+
+  def down; end
+end
```

Comment:
You can use `up_only` inside `change` to avoid the empty `down` method. 

You can use heredoc syntax for the multi-line strings, e.g:

``` ruby
execute <<~SQL
  UPDATE petitions ...
SQL
```

You can use `IN` instead of the multiple `OR` statements, e.g:

``` sql
WHERE state IN ('pending', 'validated', ...) AND collect_signatures = false
```